### DISCUSSION TITLE
How can I switch tab by recently used ones instead of cycling through all of thems

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [wezterm/wezterm#5794] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

Hey ! Like on MacOS, Arc Browser, VSCode I wanted to know if it's possible to change the tab switching mode to cycle recently used tabs instead of tabs in orders?

**Comment by wez** [comment]
https://wezfurlong.org/wezterm/config/lua/keyassignment/ActivateLastTab.html

**Reply by MatteoGauthier** [reply]
Thanks for your reply, but I'm looking for a way to cycle like this through my recent tabs, not only the latest one (cf screen record). In VSCode there is a small pane that's opening to let the user select from recent tabs

https://github.com/wez/wezterm/assets/32040951/9a068fd6-ce39-438d-8b28-8131ace4bcf9

**Reply by wez** [reply]
wezterm doesn't remember a full recency order, only the prior item.
